app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt

```diff
@@ -148,7 +147,7 @@ class StoryActivityTest {
   }
 
   @Test
-  @DisableAccessibilityChecks // TODO(#3362): Enable AccessibilityChecks
+  // TODO(#3362): Enable AccessibilityChecks
```

Comment:
If you removing `DisableAccessibilityChecks ` it means that the A11Y check is enabled and therefore TODO is not required.

Reply:
Use #3245 for all `DisableAccessbilityCheckes ` annotations